### PR TITLE
feat(UI): Add widget clock/timer

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -17963,3 +17963,40 @@ msgstr "Você não salvou as alterações, deseja continuar?"
...
+
+# msgid "Timer"
+# msgstr ""
+
+# msgid "Show time zone"
+# msgstr ""
+
+# msgid "Show date"
+# msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:18002]: @coderabbitai Stop providing translations

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -17515,3 +17515,40 @@ msgstr "Modo avanzado"
...
+
+# msgid "Select time zone"
+# msgstr ""
+
+# msgid "Select time format"
+# msgstr ""
+
+# msgid "12 hours"
+# msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17527]: @coderabbitai Stop providing translations

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -17515,3 +17515,40 @@ msgstr "Modo avanzado"
...
+# msgid "24 hours"
+# msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17530]: @coderabbitai Stop providing translations

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -17515,3 +17515,40 @@ msgstr "Modo avanzado"
...
+# msgid "Ends at"
+# msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17533]: @coderabbitai Stop providing translations

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -17515,3 +17515,40 @@ msgstr "Modo avanzado"
 
 #  msgid "You don't have sufficient permissions for this action"
 #  msgstr ""
+
+# msgid "Select time zone"
+# msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17521]: @coderabbitai Stop providing translations

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -19371,3 +19371,39 @@ msgstr "Vous ne disposez pas de droits suffisants pour cette action"
 
 msgid "Error while searching for additional connector configurations"
 msgstr "Erreur lors de la recherche de configurations additionnelles de connecteurs"
+
+msgid "Select time zone"
+msgstr "Sélectionner un fuseau horaire"
+
+msgid "Select time format"
+msgstr "Sélectionner un format de temps"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:19379]:
```suggestion
msgstr "Sélectionner un format d'heure"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -19371,3 +19371,39 @@ msgstr "Vous ne disposez pas de droits suffisants pour cette action"
...
+
+msgid "12 hours"
+msgstr "12 heures"
+
+msgid "24 hours"
+msgstr "24 heures"
+
+msgid "Ends at"
+msgstr "Fini à"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:19388]:
```suggestion
msgstr "Se termine à"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -19371,3 +19371,39 @@ msgstr "Vous ne disposez pas de droits suffisants pour cette action"
...
+
+msgid "Time zone"
+msgstr "Fuseau horaire"
+
+msgid "Date format"
+msgstr "Format de date"
+
+msgid "Time format"
+msgstr "Format de temps"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:19397]:
```suggestion
msgstr "Format d'heure"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by MedWassCh — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by dmyios — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by L0th0mat1c — APPROVED** [review]

**Review by MedWassCh — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**